### DISCUSSION TITLE
SpaceCRAFT Application

### DISCUSSION THREAD

**Discussion by gsolis1221** [SimDynamX/SpaceTeamsPRO#17] [2023 Mars City Design Competition Q&A] [answered]
Good afternoon,

I have two questions:

First, I downloaded the application and verified my account. Now that I have restarted and ran the application again to Login as a member and choose the team (unpaid) or personal (unpaid); the application freezes. 

Should I be using the SpaceCRAFT_DX11? 

Is it possible to design without buying a license or is a license required?

**Comment by connorjak** [comment, marked as answer]
- For the mars city design competition, we are managing licenses for orgs directly with accepted teams; your team leader should be in contact with us (support@simdynamx.com) to handle that. You won't need to buy a license to participate.
- I am unsure what the problem could be with the application freezing; it should allow you to try the demos without a paid account / org. Please send some screenshots of what you see and any error messages that come up.
